update-dois/src/main/java/org/reactome/release/updateDOIs/Main.java

```diff
@@ -12,55 +13,62 @@
     String pathToReport = "src/main/resources/UpdateDOIs.report";
+	  
+    boolean testMode = true;
     if (args.length > 0 && !args[0].equals(""))
     {
       pathToConfig = args[0];
-      pathToReport = args[1];
+      if (args.length > 1 && !args[1].equals("")) {
+        pathToReport = args[1];
+        testMode = false;
+      }
     }
 
     MySQLAdaptor dbaTestReactome = null;
     MySQLAdaptor dbaGkCentral = null;
     long authorIdTR = 0;
     long authorIdGK = 0;
-    boolean testMode = true;
 
     // Properties file should contain information needed to access current Test Reactome and GKCentral databases
+    String releaseNumber = "";
     try 
     {
       Properties props = new Properties();
       props.load(new FileInputStream(pathToConfig));
 
-      String userTR = props.getProperty("userTR");
-      String userGK = props.getProperty("userGK");
-      String passwordTR = props.getProperty("passwordTR");
-      String passwordGK = props.getProperty("passwordGK");
-      String hostTR = props.getProperty("hostTR");
-      String hostGK = props.getProperty("hostGK");
-      String databaseTR = props.getProperty("databaseTR");
-      String databaseGK = props.getProperty("databaseGK");
-      authorIdTR = Integer.valueOf(props.getProperty("authorIdTR"));
-      authorIdGK = Integer.valueOf(props.getProperty("authorIdGK"));
-      int port = Integer.valueOf(props.getProperty("port"));
-      if (props.getProperty("testMode") != null) {
-        testMode = Boolean.valueOf(props.getProperty("testMode"));
-      }
+      String userTR = props.getProperty("release.database.user");
+      String userGK = props.getProperty("curator.database.user");
+      String passwordTR = props.getProperty("release.database.password");
+      String passwordGK = props.getProperty("curator.database.password");
+      String hostTR = props.getProperty("release.database.host");
+      String hostGK = props.getProperty("curator.database.host");
+      String databaseTR = props.getProperty("release_current.name");
+      String databaseGK = props.getProperty("gk_central.name");
+      authorIdTR = Integer.valueOf(props.getProperty("personId"));
+      authorIdGK = Integer.valueOf(props.getProperty("personId"));
+      int portTR = Integer.valueOf(props.getProperty("release.database.port"));
+      int portGK = Integer.valueOf(props.getProperty("curator.database.port"));
+      releaseNumber = props.getProperty("releaseNumber");
+      //if (props.getProperty("testMode") != null) {
+      //  testMode = Boolean.valueOf(props.getProperty("testMode"));
+      //}
 
       // Set up db connections.
-      dbaTestReactome = new MySQLAdaptor(hostTR, databaseTR, userTR, passwordTR, port);
-      dbaGkCentral = new MySQLAdaptor(hostGK, databaseGK, userGK, passwordGK, port);
+      dbaTestReactome = new MySQLAdaptor(hostTR, databaseTR, userTR, passwordTR, portTR);
+      dbaGkCentral = new MySQLAdaptor(hostGK, databaseGK, userGK, passwordGK, portGK);
```

Comment:
Since the purpose of most of these variables is to create a MySQLAdaptor on lines 64 and 65, it might be worth creating a private method to do the extraction from the `props` object and pass in the value "release" or "curator" to extract the correct value from the Jenkins prop file for the calls to create a release and curator dba, respectively.

```
# Where databaseType is checked to be either "release" or "curator", respectively
# If you're feeling very ambitious, you could make databaseType into an Enum with values "release" and "curator" instead of a String so only permissible values are able to be passed.
private MySQLAdaptor createDBAdaptor(Properties props, String databaseType)  {

}
```

Reply:
Yea, to be honest I think updateDOIs is in need of its own refactoring, both to refresh variable naming and just to take a hard look at how its doing what its doing. Since it was so early in my rewrite days, the codes not very efficiently done nor is it apparent to the user why its doing what its doing. When I have time to do that, I'll create a seperate PR for stuff like that.

 That being said, if I can still create that function in this PR

Reply:
Great, thanks!  Could you create a GitHub issue for this separate PR, too?

Reply:
Done